seatunnel-core/seatunnel-core-base/src/main/java/org/apache/seatunnel/config/command/CommandSparkArgs.java

```diff
@@ -43,7 +43,7 @@
 
     @Parameter(names = {"-t", "--check"},
         description = "check config")
-    private boolean testConfig = true;
+    private boolean testConfig = false;
```

Comment:
Fixed the default value of `check config`, please review this again when free, thx. @leo65535

Reply:
thanks, will check the patch today again.